prometheus/base/alerting/prometheus.controller_alerts.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
+  name: o11y-controller-alerting-rules
+  namespace: o11y
+spec:
+  groups:
+  - name: controller_alerts
+    rules:
+    - alert: ControllerReconciliationErrors
+      expr: |
+        sum(increase(controller_runtime_reconcile_errors_total{namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"}[2m]))
+        by(container, controller, namespace, pod, service) /
+        sum(increase(controller_runtime_reconcile_total{namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"}[2m]))
+        by(container, controller, namespace, pod, service) * 100 > 1
```

I think it's better to eliminate the multiplication here
```suggestion
        by(container, controller, namespace, pod, service) > 0.01
```